build.csx

```diff
@@ -61,7 +61,7 @@ static var testsDirectory = "./artifacts/tests";
 // targets
 var targets = new TargetDictionary();
 
-targets.Add("default", DependsOn("pdbgit", "unit", "integ", "spec", "approve", "pack"));
+targets.Add("default", DependsOn("unit", "integ", "spec", "approve", "pdbgit", "pack"));

```

Comment:
An alternative would be to have pack depend on pdbgit, especially if our model is "there's no point in packing something that hasn't been pdbgitted".